src/core/QueryManager.ts

```diff
@@ -1296,6 +1296,13 @@ export class QueryManager {
 
     if (include) {
       this.getObservableQueries(include).forEach((oq) => {
+        if (
+          (include === "active" || include === "all") &&
+          oq.options.fetchPolicy === "cache-only"
+        ) {
+          return;
+        }
+
```

Comment:
```suggestion
        if (oq.options.fetchPolicy === "cache-only") {
          return;
        }

```

As a result of my previous comment, this should probably do it.